internal/pkg/encoding/jsonnet/vm.go

```diff
@@ -16,16 +16,16 @@ type VM[T any] struct {
 	lock    sync.Mutex
 	vm      *jsonnet.VM
 	err     error
-	payload *T
+	payload T
 }
 
 // Payload returns the payload of the current evaluation.
 // Can be used by custom functions for request specific values.
-func (vm *VM[T]) Payload() *T {
+func (vm *VM[T]) Payload() T {
```

Comment:
I need here INTERFACE `recordctx.Context` (there is HTTP and FastHTTP implemetation).

Reply:
Maybe if it is not working well, we just switch the implementation to `HTTP` one.

Reply:
@jachym-tousek-keboola 
- `TestSource` endpoint is working with Goa/standard HTTP library, so both versions are needed.
- If some new type of Source will be implemented in the future, then another implementation will also be added.